force-app/main/default/tests/RollbarTest.cls

```diff
@@ -5,17 +5,43 @@ public class RollbarTest {
         Test.setMock(HttpCalloutMock.class, new RollbarApiCalloutMock());
 
         Rollbar.init('test-token', 'test-env');
-        HttpResponse response = Rollbar.log('info', 'Message from the Apex SDK');
+        HttpResponse response = Rollbar.log('info', 'Message from the Apex SDK', SendMethod.SYNC);
 
         System.assertEquals(200, response.getStatusCode());
     }
 
+    @isTest
+    public static void testLogMessageFuture() {
+        Test.setMock(HttpCalloutMock.class, new RollbarApiVerifyRequestCalloutMock());
+
+        insert new RollbarSettings__c(AccessToken__c = 'test-token');
+
+        Test.startTest();
+        Rollbar.log('info', 'Message from the Apex SDK', SendMethod.FUTURE);
+        Test.stopTest();
+
+        // Asserts valid payload in the mock.
```

Comment:
This looks like a no-opp unit test.. I don't fully understand the logic here, but I would at least confirm the data payload is correct

Reply:
Because of the way future methods execute, the asserts have to be in the mock. That's why the comment is there.